input_output.ipynb

```diff
@@ -465,7 +500,15 @@
    "outputs": [],
    "source": [
     "import tutorial.input_output as op\n",
-    "op.Paths()"
+    "op.Paths()\n",
+    "from pathlib import Path\n",
+    "\n",
+    "\n",
+    "current_path = Path(\"tutorial/tests/data/\")\n",
+    "print(list(current_path.iterdir()))\n",
+    "#Output all file names in that directory\n",
+    "for file in current_path.iterdir():\n",
+    "    print(file.name)"
```

Comment:
Why is this code here? When executing the cell, the quiz does not appear

Reply:
My mistake